### PR TITLE
Remove Python 2.7 and 3.5

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

https://github.com/computationalmodelling/nbval/issues/160

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,15 +15,10 @@ install:
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
   - unset VIRTUAL_ENV
-  # Python 3.4 and 2.7 need six installed before running setup.py
-  - pip install six
   # Travis needs to update pip/setuptools
   - pip install --upgrade setuptools pip
   - pip install .
   - |
```
Comment (takluyver) [inline .travis.yml:21]:
```suggestion
```

This is a marker for the block on the lines below.

**Review by takluyver — COMMENTED** [review]
LGTM, with one small tweak:

File: .travis.yml (hunk 1 above)
Comment (gliptak) [reply .travis.yml]: updated

**Comment by vidartf** [comment]
We should probably remove any py2 specific bits from the codebase too then, i.e. any use of `six`.

**Comment by gliptak** [comment]
@vidartf removed six

**Comment by vidartf** [comment]
Thanks. This code in plugin.py is also py2 specific, I think that would be the last bits:

```python
from __future__ import print_function

# ... and

try:
    from Queue import Empty
except:
    from queue import Empty
```

(https://stackoverflow.com/questions/56047277/python-queue-vs-queue)

File: nbval/plugin.py [hunk 2]
```diff
@@ -16,13 +16,7 @@
...
 from collections import OrderedDict, defaultdict
 
-# for python 3 compatibility
-import six
-
-try:
-    from Queue import Empty
-except:
-    from queue import Empty
+from Queue import Empty
```
Comment (takluyver) [inline nbval/plugin.py:19]:
```suggestion
from queue import Empty
```

The lowercase name is the Python 3 variant.

Comment (gliptak) [reply nbval/plugin.py]: updated. thx

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**